opendocdb-cts/main.go

```diff
@@ -106,7 +111,7 @@ func convertCommand() error {
 		for tcName, tc := range ts {
 			filename := fmt.Sprintf("%s.js", tcName)
 
-			req, res, err := convertTestCase(tc)
+			req, res, err := mongosh.ConvertTestCase(tc)
```

Comment:
Note how `internal/data` provides functions that operate on whole directory structures, not just files. We should design `internal/mongosh` in a similar way. `convertCommand` should become even smaller.